text_collector_examples/multipathd_info

```diff
@@ -0,0 +1,9 @@
+#/bin/bash
+#
+# Description: Expose device mapper multipathing metrics from multipathd.
+#
+# Author: Saket Sinha <saket.sinha@cloud.ionos.com>
+
+echo '# HELP node_dmpath_info State info for dev-mapper path'
+echo '# TYPE node_dmpath_info gauge'
+/sbin/multipathd show paths | /usr/bin/awk '{ if ( NR > 1  ) {print "node_dmpath_info{device=\""$2"\"," "path_state=\""$5"\"," "dm_path_state=\""$6"\"}" " 1"}}'
```

Comment:
I'm not very familiar with the output of this, but it seems like sometimes the output makes the columns squashed together.

If you look at [this](https://access.redhat.com/solutions/17303), the columns are squished together for `[active]`.

Reply:
Sorry, I cant access [this](https://access.redhat.com/solutions/17303).

On my system the output looks like something below -

ssinha@ps101a-102:~ sudo multipathd show paths
hcil      dev  dev_t   pri dm_st  chk_st dev_st  next_check      
4:0:0:1   sdc  8:32    1   active ready  running X......... 7/40 
5:0:0:1   sde  8:64    1   active ready  running XX........ 11/40
8:0:0:2   sdl  8:176   1   active ready  running XXXXXXXXX. 36/40

ssinha@ps101a-102:~ sudo /sbin/multipathd show paths | /usr/bin/awk '{ if ( NR > 1  ) {print "node_dmpath_info{device=\""$2"\"," "path_state=\""$5"\"," "dm_path_state=\""$6"\"}" " 1"}}'
node_dmpath_info{device="sdc",path_state="active",dm_path_state="ready"} 1
node_dmpath_info{device="sde",path_state="active",dm_path_state="ready"} 1
node_dmpath_info{device="sdl",path_state="active",dm_path_state="ready"} 1

Let me know if you find something wrong with this.

Reply:
Ok, I guess that version has different output compared to the other one. Seems OK.